src/testing/localStorage.test.js

```diff
@@ -0,0 +1,44 @@
+describe('Test for add and delete functions', () => {
+  test('Test for add function', () => {
+    // Arrange
+    let library;
+    if (!library) {
+      library = [];
+    } else {
+      library = JSON.parse(localStorage.getItem('library'));
+    }
+    const content = {
+      completed: false,
+      task: 'kvnodfl',
+      index: 1,
+    };
```

Comment:
**[OPTIONAL]**

- Great job with the test. I think your codebase isn't dry and you're repeating yourself, since you're using the same content and pushing it into the array, you could actually move these blocks of code immediately after the describe block so you could re-use them in each test...